.github/dockerimages/Dockerfile.macos

```diff
@@ -0,0 +1,76 @@
+# Use the latest macOS compatible image (experimental)
+FROM bot62726170686f67/macos-cross-rust:latest
```

Comment:
This image is quite old, can we use a newer one? Are we using Rust for anything here? Or will we need it in the future?